x-pack/plugins/endpoint/public/embeddables/resolver/store/data/reducer.ts

```diff
@@ -42,7 +42,7 @@ export const dataReducer: Reducer<DataState, ResolverAction> = (state = initialS
     if (statsMap) {
       const currentStatsMap = new Map(statsMap);
       const resolverEvent = action.payload;
-      currentStatsMap.set(resolverEvent, new Error('error requesting related events'));
```

Comment:
:straight_ruler: Don't: Use `Error`s to represent errors. Do: Use a string or an object